images/openshift-ci/Dockerfile

```diff
@@ -4,15 +4,15 @@ FROM scratch AS builder
 WORKDIR /code-ready/snc
 COPY . .
 
-FROM centos:7
+FROM centos:8
```

Comment:
CentOS 8 is going EOL in a month. Did we test with
```
dnf swap centos-linux-repos centos-stream-repos
```
if CetnOS Stream works?

Reply:
@gbraad I think those are container images so it should automatic have centos stream repos.

The container created from this repo have the app stream repo so we are good.
```
# dnf repolist
repo id                                                                                   repo name
appstream                                                                                 CentOS Linux 8 - AppStream
baseos                                                                                    CentOS Linux 8 - BaseOS
extras                                                                                    CentOS Linux 8 - Extras
google-cloud-sdk                                                                          Google Cloud SDK
```